bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -4,6 +4,8 @@
 
 {% extends "firefox/base-resp.html" %}
 
+{% add_lang_files "firefox/new/horizon" %}
```

Comment:
@flodolo including the lag file here from the download page for "Browse Freely". Assuming this is the easiest (best?) solution.